spec/models/measurement_spec.rb

```diff
@@ -36,4 +36,10 @@
       end
     end
   end
+
+  describe '#approximate_count' do
+    it 'shoulde return approximate count' do
```

Comment:
```suggestion
    it 'should return approximate count' do
```
OR just
```
 it 'returns approximate count' do
```
More info: http://www.betterspecs.org/#should